shortages-prediction-adapters/src/test/groovy/io/dddbyexamples/factory/shortages/prediction/monitoring/persistence/Config.groovy

```diff
@@ -0,0 +1,45 @@
+package io.dddbyexamples.factory.shortages.prediction.monitoring.persistence
+
+import groovy.transform.CompileDynamic
+import groovy.transform.CompileStatic
+import org.mockito.Mockito
+
+import org.springframework.beans.BeansException
+import org.springframework.beans.factory.config.BeanPostProcessor
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+
+/**
+ * @author Marcin Grzejszczak
+ */
+@Configuration
+@EnableAutoConfiguration
+@ComponentScan("io.dddbyexamples.factory.shortages.prediction.persistence")
+@CompileStatic
+class Config {
+
+	// https://github.com/spring-projects/spring-boot/issues/7033
+	@Bean
+	BeanPostProcessor shortagesBeanPostProcessor() {
+		return new BeanPostProcessor() {
+			@Override
+			Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
+				if (bean instanceof ShortagesDao) {
+					return Mockito.mock(ShortagesDao)
```

Comment:
hmm, why?

Reply:
Because we want to mock out the access to the real database - https://github.com/spring-projects/spring-boot/issues/13298